### PR TITLE
Update to use latest travel-sample bucket and frontend

### PR DESCRIPTION
**Pull request by chvck**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    build: "https://github.com/couchbaselabs/try-cb-frontend-v2.git#7.0"
+    ports:
+      - 8081:8081
+    container_name: try-cb-fe
+    entrypoint: ["wait-for-it", "backend:8080", "--timeout=400", "--strict", "--", "npm", "run", "serve"]
+    depends_on:
+      - backend
+
+  db:
+    image: build-docker.couchbase.com:443/couchbase/server-internal:7.0.0-5247
```
Comment (maria-robobug) [inline docker-compose.yml:22]: I guess for testing purposes we'd use the internal image, but probably we can change this to the sandbox image when 7.0 GA is publicly available? And maybe for now leave it as `couchbase/server-sandbox:7.0.0-beta`
 
 https://github.com/couchbase/server-sandbox/tree/7.0.0-beta

File: main.go [hunk 2]
```diff
@@ -555,82 +572,168 @@ func HotelSearch(w http.ResponseWriter, req *http.Request) {
...
-		res.ContentAt(5, &hotel.Description)
+		err = res.Content(&hotel)
+		if err != nil {
+			app.logger.Printf("Error occurred parsing hotel: %s", err)
+			writeJsonFailure(w, 500, err)
+			return
+		}
 		respData.Data = append(respData.Data, hotel)
 	}
+	b, err := json.Marshal(qp)
```
Comment (maria-robobug) [inline main.go]: Find it difficult to tell what `b` represents, maybe we could change to a more explicit variable name?

File: main.go [hunk 3]
```diff
@@ -555,82 +572,168 @@ func HotelSearch(w http.ResponseWriter, req *http.Request) {
...
+	scheme := envFlagString("CB_SCHEME", "scheme", defaultCBScheme,
+		"The scheme to use for connecting to couchbase. Default to couchbases - set to couchbase:// when using couchbase community edition")
+	flag.Parse()
+
+	logrusLogger := logrus.New()
+	logrusLogger.SetFormatter(&logrus.JSONFormatter{})
+	logrusLogger.SetOutput(os.Stdout)
+	logrusLogger.SetLevel(logrus.InfoLevel)
+
+	// Uncomment to enable the Go SDK logging.
```
Comment (maria-robobug) [inline main.go:654]: Why not enable this anyway? Or would it make the logs too noisy when running the server?

**Review by maria-robobug — APPROVED** [review]
Nice refactoring of the app and yai for getting swagger working, 😃  - LGTM,  just have some minor questions/comments!

File: swaggerui/index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        margin:0;
+        background: #fafafa;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div id="swagger-ui"></div>
+
+    <script src="./swagger-ui-bundle.js" charset="UTF-8"> </script>
```
Comment (maria-robobug) [inline swaggerui/index.html:36]: Question more for my benefit/learning! How did you manage to get all the necessary swagger-ui files?

File: docker-compose.yml (hunk 1 above)
Comment (chvck) [reply docker-compose.yml:22]: Makes sense however it won't actually work with the beta version anyway.

File: main.go (hunk 3 above)
Comment (chvck) [reply main.go:654]: Yeah it adds a whole load of logging that I don't think users will actually be interested in unless they need to debug something.

File: swaggerui/index.html (hunk 4 above)
Comment (chvck) [reply swaggerui/index.html:36]: I nabbed the dist folder contents from https://github.com/swagger-api/swagger-ui

**Pull request merged**